packages/khalti/pubspec.yaml

```diff
@@ -1,21 +1,25 @@
 name: khalti
-description: An official Flutter plugin for Khalti Payment Gateway, to accept payments using Khalti Wallet, E-banking, Mobile Banking, Connect IPS and SCT cards.
+description: An official Flutter plugin for Khalti Payment Gateway, to accept
+  payments using Khalti Wallet, E-banking, Mobile Banking, Connect IPS and SCT
+  cards.
 version: 1.0.2
 homepage: https://khalti.com/payment-gateway/
 documentation: https://docs.khalti.com/
 issue_tracker: https://github.com/khalti/khalti-flutter-sdk/issues
 repository: https://github.com/khalti/khalti-flutter-sdk/tree/master/packages/khalti
 
 environment:
-  sdk: '>=2.12.0 <3.0.0'
+  sdk: ">=2.17.6 <3.0.0"
 
 dependencies:
+  device_info_plus: ^4.0.2
   flutter:
     sdk: flutter
-  device_info_plus: ^3.2.1
-  http: ^0.13.4
+  http: ^0.13.5
   khalti_core: ^1.0.1
-  package_info_plus: ^1.3.0
+  mocktail: ^0.3.0
+  package_info_plus: ^1.4.3
+  plugin_platform_interface: ^2.1.2
 
 dev_dependencies:
-  flutter_lints: ^1.0.4
\ No newline at end of file
+  flutter_lints: ^1.0.4
```

Comment:
device_info_plus:v4 uses flutter_lints:v2, isn't pub get complaining about it ?

Reply:
No complaints at all.